lib/tasks/reset.rake

```diff
@@ -0,0 +1,2 @@
+desc 'Recreate the database and set up all of the test data'
+task reset: %i[db:reset db:migrate setup_local_dev_data generate_test_applications]
```

Comment:
`local_dev_data` calls `GenerateTestApplications` already

Reply:
It does not!

Reply:
Narrator: _it did_